draft-ietf-quic-transport.md

```diff
@@ -4115,6 +4130,25 @@ preferred_address (0x000d):
 ~~~
 {: #fig-preferred-address title="Preferred Address format"}
 
+version_aliases (0x000e):
+
+: A list of version numbers that the server accepts as an alias for the
+  currently used versions. This transport parameter is only sent by the server.
+  Every version alias contains a lifetime in seconds. The alias is only valid
+  for that lifetime, clients MUST NOT use it after expiry.
+
+~~~
+   struct {
+     uint32 VersionNumber;
+     varint Lifetime;
```

Comment:
`varint` is not part of the TLS Presentation Language.

Reply:
Right. I was hoping for some help from people more familiar with the TLS presentation language on how to write it. We're using varints for all the other transport parameters, but we successfully managed to never actually use presentation language.